pkg/kubenest/controller/global.node.controller/global_node_lifecycle_controller.go

```diff
@@ -105,6 +124,25 @@ func (c *GlobalNodeStatusController) updateStatusForGlobalNode(
 			statusType = "NotReady"
 		}
 
```

Comment:
o ensure code reuse and consistency, you should use Kubernetes-defined constants whenever possible, instead of hardcoding strings like "NotReady".